### PR TITLE
Improve teams:fetch rake task and related documentation

### PR DESCRIPTION
**Pull request by byroot**
Fix: https://github.com/Shopify/shipit-engine/issues/553

@lucasmazza does that looks better to you?

### CODE REVIEW

File: lib/tasks/teams.rake [hunk 1]
```diff
@@ -0,0 +1,9 @@
+namespace :teams do
+  desc "Import a team and it's members from Github and install a webhook to monitor it"
```
Comment (lucasmazza) [inline lib/tasks/teams.rake]: s/Github/GitHub/

**Comment by lucasmazza** [comment]
@byroot looks awesome, thanks :heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

File: lib/tasks/teams.rake (hunk 1 above)
Comment (lucasmazza) [inline lib/tasks/teams.rake]: "Import a team and it's members" could be "Import the members of each team configured through the `github_oauth.teams` config" ?

**Comment by byroot** [comment]
I improved it a bit further, mind giving a second look please?

**Comment by lucasmazza** [comment]
@byroot perfect :+1:

**Pull request merged**